didx-xyz/yoma-mobile
# YOMA-523 - feat(Education): show all user qualifications in CV view

## Scope [YOMA-523](https://yomaworld.atlassian.net/browse/YOMA-523)

![](https://media.giphy.com/media/1iTpx5PpzRugcrZK/giphy.gif)

## Work Done
- adding selectors to handle getting user qualifications from state
- updating Education module to handle view separately to form (as with Completed Challenges), adding selector for View to handle getting the user qualifications

- refactored CompletedChallenges so that form is in its own component, and view is in its own component and CompletedChallenges.tsx only handles toggling the views.

## Steps to Test
run tests
View in app - make sure Education view (click on View all in Education widget) is loading real data.
    
## Screenshots
<img width="451" alt="Screenshot 2021-09-07 at 21 26 54" src="https://user-images.githubusercontent.com/1424561/132400096-c97b77a8-8961-45b0-8500-f2da011d12cf.png">
